src/wiiu/wiiu.c

```diff
@@ -216,6 +215,8 @@ void wiiu_error_exit(char* fmt, ...)
 
   wiiu_screen_draw();
 
+  // Re-enable home button detection for exit
+  WHBProcInit();
```

Comment:
I don't think initializing ProcUI again is a good idea.
Maybe re-registering the home button callback which calls `WHBProcStopRunning` would work?